packages/imperative/CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed issue in local web help with highlighted sidebar item getting out of sync. [#2215](https://github.com/zowe/zowe-cli/pull/2215)
+- BugFix: Updated web help dependencies for technical currency. [#2215](https://github.com/zowe/zowe-cli/pull/2215)
+
 ## `5.26.1`
 
-- Bugfix: Export new Proxy class from Zowe imperative package. [#2205](https://github.com/zowe/zowe-cli/pull/2205)
+- BugFix: Export new Proxy class from Zowe imperative package. [#2205](https://github.com/zowe/zowe-cli/pull/2205)
```

Comment:
How is this a bug fix? Why did this need to be addressed?

Reply:
Good catch, this is a release note from a previous PR but I'll update it to match what we have on the vNext branch:
> - BugFix: Fixed missing export for `Proxy` class in Imperative package.

Pushed the update on another PR: https://github.com/zowe/zowe-cli/pull/2218